Tasks/ServiceFabricDeploy/deploy.ps1

```diff
@@ -102,8 +102,15 @@ try {
     $useDiffPackage = [System.Boolean]::Parse((Get-VstsInput -Name useDiffPackage))
     if ($useDiffPackage)
     {
-        Import-Module "$PSScriptRoot\Create-DiffPackage.psm1"
-        $diffPackagePath = Create-DiffPackage -ApplicationName $applicationName -ApplicationPackagePath $applicationPackagePath -ConnectedServiceEndpoint $connectedServiceEndpoint -ClusterConnectionParameters $clusterConnectionParameters
+        if (Test-ServiceFabricApplicationPackage -ApplicationPackagePath $applicationPackagePath)
```

Comment:
We should ignore Test-ServiceFabricApplicationPackage if skipValidation is set to true